.gitignore

```diff
@@ -16,3 +16,5 @@
 
 # Ignore .DS_store file
 .DS_Store
+
+Gemfile.lock
```

Comment:
This file shouldn't be locked/removed. We want to ensure that every one deploying the website has the exact same version of dependencies.